## About the MyConnectome Project

The MyConnectome project aimed to characterize how brain function and metabolism fluctuate over the course of an entire year in a single individual.

This study was motivated by an increasing interest in understanding the dynamics of brain function over a days-months timescale and how they relate to cognitive function and bodily metabolism.  At the time of the study, there were basically no data in existence that provide any insight into how the function of an individual’s brain fluctuates over such a relatively long time course. This is probably not surprising, because doing studies with volunteers that require repeated testing over a long period of time is very challenging.  

In 2011, Dr. Poldrack began working with a team of researchers at UT and beyond to design a study that would collect such a dataset from himself.  There were several inspirations for this idea.  First was Michael Snyder’s “integrated personal omics” study, published in Cell in 2011, in which he repeatedly collected blood from himself and performed a broad set of “omics” measures on his samples, which provided some interesting insights into the temporal dynamics of metabolic function.  Second was his interaction with Laurie Frick, who is the artist-in-residence at the UT Imaging Research Center.  Laurie’s work is based on patterns that she finds in data obtained by self-tracking, and she is deeply enmeshed in the Quantified Self movement (see her excellent TEDx talk).  Talking to her got him increasingly interested in tracking a broader set of data about himself.  He was also inspired by other self-experimentalists, both historical and current.

It was essential that the all aspects of data collection were as consistent as possible in order to minimize extraneous variability in the data (such as time of day effects).  We settled on a schedule of three MRI scanning sessions a week, at consistent times of day and days of the week (one afternoon and two mornings every week).  Each of the MRI scanning sessions includes a resting state fMRI scan, which will allow us to assess how functional connectivity between brain regions fluctuates over time.  In addition, once a week we performed other scans, including structural MRI (T1- and T2-weighted), diffusion tensor MRI (to assess white matter connectivity), and task fMRI (using a working memory task with faces, scenes, and chinese characters).

We also collected biological samples in order to measure the relation between bodily metabolism and brain function.  Working with some molecular biologists here at UT (along with helpful input from the Snyder lab at Stanford), we developed a protocol in which I had 20 ml of blood drawn once a week (while fasting, immediately after one of the morning MRI scans). This sample was then processed to extract RNA, white blood cells, and plasma, all of which were frozen for later analysis.  This let us examine many different aspects of metabolism, including gene expression (via RNA sequencing) and metabolomic analyses.

The project also involved collecting as much data as possible about daily life activities.  Working with Zack Simpson, we developed a self-tracking app using the Appsoma framework, which allows completion of surveys every morning and evening and after every MRI scan.  These data are automatically fed into a web database which is the central repository for all of the self-tracking data in the study other than MRI and biological analyses.  Some of the things that are tracked daily include:

- blood pressure and weight
- foods eaten, alcohol intake, and supplements/medicines taken
- exercise, time spent outdoors, and physical soreness
- a free-text log of daily events

The assessment after every scanning session included a mood questionnaire and a structured report of what the subject was thinking about during the resting state fMRI scan.

With this plan in place, we began data collection on September 25, 2012.  We treated the first month as a pilot period, and made some changes to the imaging protocol to optimize data collection, beginning the production period on October 22, 2012.  The final regular data collection session was collected on March 11, 2014; a number of scan sessions have been collected in the subsequent decade.

For details regarding the data acquisition and analysis, visit the [Detailed Protocol](detailed-protocol.md) page.

 