
### Detailed Study Protocol

#### Pre-screening

Prior to initiation of the pilot period, the participant received a full blood workup (CBC, CMP, TSH, ESR, and lipid panel) along with tests for HIV and Hepatitis B and C.  No significant findings were observed in these tests.

#### Pilot phase

A pilot period began on 9/25/2012. During the pilot phase, the initial protocol was tested for several weeks.  Upon examination of the data and further discussion with a number of other researchers, several changes were made to the MRI acquisition during the week of 10/15/2012.

#### Production phase

The production phase began on 10/22/2012 (with session self-tracking\_0013).

#### Audiometry

During the pilot period, the participant noticed an increase in the level of his pre-existing tinnitus.  Because of the potential for noise-induced hearing damage, baseline audiometry was performed on 10/16/2012 (within 8 hours of a scanning session), 10/17/2012 and 10/19/2012 (both ~26 hours after previous scanning session).  Tympanometry and reflex threshold testing were performed at the first exam, and were normal.  These exams showed moderate hearing loss (40-60 dB) in the high-frequency range (at or above 3 kHz) which was consistent across the two sessions.  In addition, there was a slight (10-15 dB) loss in the lower frequencies which was reduced by ~ 5 dB in the later tests, suggesting a potential short-term effect.  However, this could also have reflected the resolution of seasonal allergies across the testing days.  Distortion product otoacoustic emission (DPOAE) testing on the first testing day showed consistent results, with impaired function at 3 and 4 kHz.   Audiometry will be repeated every month to ensure that no further hearing damage occurs.

On March 6, 2013, audiometry showed increased hearing loss, up to 70 dB at 6000 Hz.  For this reason, imaging was temporarily stopped.  Followup testing was performed on May 1, which showed that the apparent decrease was not sustained.  The protocol was re-started on April 30 with a more limited set of measurements

#### Subject blinding

In order to prevent knowledge of the results from affecting the subject, the subject was blinded to the results of any analyses of the repeated tests during the first six months of data collect; he had access to the clinical blood work results, and also examined individual MRI scans for quality control purposes, but was not be exposed to any analysis of temporal changes.

Research staff ensured quality control of the data as they were collected, as detailed below, but did not inform the subject of any research findings during this period.

After the hiatus in mid-March, this blinding was broken and the subject analyzed the first set of data collected to that point.  The blinding will continue during the period of continued data collection, such that new data (collected after April 30, 2013) will not be examined by the subject.

#### MRI

MRI imaging will be performed on a fixed schedule, subject to availabilty of the participant.  Scans on Monday will be performed at 5 pm.  Scans performed on Tuesday and Thursday will be performed at 7:30 am.  After the hiatus in mid-March, it was decided to reduce this to twice a week, on Tuesdays and Thursdays, though Monday sessions will be performed when possible.

_Scans to be performed every session:_

**_Localizer_**: Standard 3-plane and AutoAlign localizers will be performed for each run.

_Resting state fMRI_.  RS-fMRI will be performed using a multi-band EPI (MBEPI) sequence (TR=1.16 ms, TE = 30 ms, flip angle = 63 degrees, voxel size = 2.4 mm X 2.4 mm X 2 mm, distance factor=20%, 68 slices, oriented 30 degrees back from AC/PC, 96×96 matrix, 230 mm FOV, MB factor=4, 10:00 scan length).  Starting with session 0027 (12/3/2012), the number of slices was changed to 64 because of an update to the multiband sequence which increased the minimum TR beyond 1.16 for 68 slices.

Noise cancellation for the resting-state scan was attempted in each session, but in some cases it failed to work.  This is recorded in the scanning notes for each scanning run (stored in RedCap).

_Field maps_: A gradient echo field map sequence will be acquired with the same prescription as the functional images.  In addition, spin echo field maps are collected with A-P and P-A phase encoding. NOTE: Field maps were discontinued as of April 30, 2013.

_Scans to be performed once weekly:_

_Task fMRI (n-back)_:  An n-back task will be performed using a blocked design, with a factorial combination of memory load (1 vs. 2 back) and stimulus type (faces, houses, and chinese characters) across blocks.  20% of items will be targets, and 20% will be foils.  Stimuli will not be repeated across sessions.  At the end of the session, the subject will perform a source memory task to identify which stimuli appeared as n-back targets (equated for familiarity with items that appeared as distractors).  The acquisition sequence will be identical to that used for the RS-fMRI scan. (acqusition time = 8:00)

**NOTE**: Regular weekly acquisition of the following anatomical and diffusion scans was discontinued on April 30, 2013.

_T1-weighted anatomy_:  An MP-RAGE sequence will be acquired using a sequence patterned after the Human Connectome Project (saggital, 256 slices, 0.7 mm isotropic resolution,, TE=2.14 ms, TR=2400 ms, TI=1000 ms, flip angle = 8 degrees, PAT=2, 7:40 scan time)

_T2-weighted anatomy_: A T2-SPACE sequence will be acquired (saggital, 256 slices, 0.7 mm isotropic resolution,, TE=565 ms, TR=3200 ms, PAT=2, 8:24 scan time)

_Diffusion-weighted imaging_: Two DWI scans will be acquired using a multiband EPI sequence, one with L->R phase encoding and the other with R->L phase encoding, for later unwarping using FSL’s topup tool.  A Stejskal-Tanner sequence will be used to optimize signal, since eddy currents can be addressed using the FSL eddy tool.  Two shells of 30 directions are acquired (b=1000 and 2000).  Four low-b images are interspersed among the 60 diffusion-weighted images (one every 15 frames).  Straight axial slices are aquired to avoid problems with the reorientation of diffusion vectors.  Parameters are b=1000/2000, 1.74 X 1.74 X 17 mm resolution, 72 slices, FOV=223 mm, 128 X 128 matrix, TR=5000 ms, TE = 108 ms, PAT=2, MB factor=3)

_Scans to be performed (at least) once during the course of the study:_

_Task fMRI_: A number of additional tasks will be run over the course of the year, including all of the tasks included in the HCP task fMRI battery.

_Retinotopy_:  Retinotopic mapping will be performed in collaboration with the Huk lab .

_Additional tissue parameter mapping_: Additional scans may be added to map particular aspects of tissue.

_HARDI_:  A multi-shell high angular resolution diffusion sequence will be performed at least once, for use in tractography.

Structural scans will be examined visually immediately following the scan; in the case of artifacts due to excessive movement, the scan may be re-run at the discretion of the operator.

#### Lifestyle/psychological measurements:

Behavior ratings.  The participant will complete behavioral ratings daily via the appsoma web portal:

  

_Measurement_

_Instrument_

**Morning**

Sleep quality

1-7 rating scale

Physical soreness

1-7 rating scale

Blood pressure

**After Each MRI Scan**

Mood

PANAS-X (all 60 questions)

Blood pressure

Anxiety during scan session

1-7 scale

thoughts during scan

free text

**Evening**

Time spent outdoors

Estimated # of minutes

Psoriasis severity

1-7 rating scale

Stress

1-7 rating scale

Gut health

1-7 rating scale

Mood

PANAS-X (short form)

Open-ended report

Free text

Report any mental or physical symptoms experienced during the day (e.g, ‘headache’ )

Diet

Free text

Report all foods eaten that day

Alcohol

Floating point

\# of standard drinks consumed

_Environmental measures_: Record time spent outdoors.  Other environmental measures can be obtained automatically (weather data via NOAA API, sunrise/sunset via PyEphem , pollen count via wunderground api).

_Exercise logging_: The participant will wear a heart rate monitor during all exercise sessions (Garmin FR70).  Data will be exported to Garmin Connect and .FIT files will also be archived on TACC. Will quantify the amount of time spent at HR > 120 and HR > 180.

_Weight_: Naked weight and body fat will be measured daily, upon waking, using the FitBit Aria scale.  The results are automatically transmitted to the FitBit site, and will be downloaded automatically every week to the study database.

_Sleep_: Sleep will be measured nightly using ZEO sleep monitor.  Data will be obtained directly via the ZEO API and downloaded automatically every week to the study database. NOTE: The Zeo API is no longer available as of May 2013.

_Blood pressure/pulse_. BP and pulse will be measured daily upon waking and immediately after each MRI scan, and recorded via the web portal.

_Diet_: Every day, a textual description of all foods eaten (including as many known ingredients as possible) will be taken, along with a recording of the amount of alcohol consumed each day.

#### Physiological measurements

Blood will be drawn every Tuesday around 8 am, following the MRI scan.  Blood draws will be performed at the UT Student Health Center Laboratory. The subject will refrain from eating after 8 pm the previous night and remain fasted until after the blood draw.

Blood draw limits: Based on the guidelines from UCLA:

1.  No more than 2.5% of total blood volume may be drawn solely for research purposes (no benefit to the subject) within a 24-hour period. This is generally 2 ml/kg.
    
2.  No more than 5% of total blood volume may be drawn solely for research purposes (no benefit to the subject) within a 30-day period. This is generally 4 ml/kg.
    

Subject must maintain a minimum Hemoglobin of 12.5 g/dl.

_Sample collection_. The standard blood draw will include:

*   1 5ml lavender EDTA tube (to be provided by UT Student Health Services Lab)
    
    *   for CBC/diff, to be performed only for the first scan of each month
        
*   2 10 ml venous blood collection tubes (lavender K+/EDTA)
    
    *   Transported immediately to UT Genomic Sequencing and Analysis Facility (GSAF)  for processing.
        

During any periods of illness and following any vaccinations, blood may be taken more often (in consultation with collaborating researchers and physician) for proteomic and other analyses.

_Sample preparation_.  Peripheral blood mononuclear cells (PBMCs) will be harvested by Ficoll gradient from whole blood collected in standard K+/EDTA blood collection tubes.  At the first and third collection, 200ul of whole blood will be used for DNA isolation.   The PBMC population, isolated by Ficoll gradient,  will then be split into two equal fractions:  one washed in PBS and frozen for future use and one immediately placed in lysis solution to eliminate nuclease activity for RNA isolation.

Whole-exome sequencing.  10 micrograms of genomic DNA were used to create next-generation sequencing libraries for exome sequencing.  The standard Illumina DNA fragment library protocol (or equivalent commercial reagents) was used to shear, end-repair, dA-tail, and ligate sequencing adaptors.  For exome sequencing, these finished libraries were enriched using the Nimblegen SeqCap kit.

_RNA-seq_: Total RNA will be extracted from collected PBMCs following Standard Operating Protocols which will be developed specifically for this study to minimize variation in gene expression due to processing.  Total RNA will then be analyzed using next-generation DNA sequencing to determine gene and non-coding RNA abundance and splice variation.  Using commercially available kits and reagents they are already very familiar with the UT GSAF will transform the RNA into sequence-ready libraries.  Briefly, the total RNA will be fragmented using elevated temperature in carefully controlled buffer conditions to yield average fragment sizes of 200 nucleotides.  These fragments will be directionally ligated to 5′ and 3′ adaptors so that sequence orientation is preserved throughout sequencing.  Reverse transcription and PCR are performed to complete the DNA sequencing libraries which will be sequenced on the HiSeq 2000 or 2500.

_Metabolics_:  Aliquots of 500 µL serum or plasma will be sent to Myriad-RBM for analysis using the DiscoveryMAP v1.0 (188 analytes).

_Serum proteomics_: Pending availability of funds, serum samples analyzed using proteomics or metabolomics approaches.

#### Data management:

All experimental data will be stored to a couchdb database, which will be backed up regularly to a  repository located on the replicated TACC Corral filesystem, in order to ensure data integrity and security.  Neuroimaging data will be stored on the XNAT archive on TACC.

_Survey data_:  All survey data are acquired using the appsoma platform, and immediately transmitted to couchdb  upon completion of the survey.  Each survey is identified by the survey variable in the database.

_DNA Sequencing_: After sequencing, the GSAF transfers data to the Texas Advanced Computing Center (TACC) where the Lonestar cluster is used for basecalling and/or separation of data based on barcode index.  Subsequently, the GSAF will process either the whole genome or exome data using the Broad GATK pipeline for human variation analysis using existing scripts also on the Lonestar cluster.  Briefly, this process includes mapping of the raw fastq sequence data to the human genome with the BWA alignment and mapping program, re-calibration of quality values and local re-alignment around any putative insertions or deletions with GATK tools, and finally genotyping with the GATK Unified Genotyper.  Genotypes will be annotated with Annovar and/or VAAST programs.

_RNA-seq_: As with the DNA sequences, raw data will be transferred to the Lonestar cluster for separation based on barcode index.  Subsequently, the UT GSAF will utilize the Tuxedo software suite to map the RNA sequences to the annotated human genome to determine exon- and gene-level expression and splice variation with the programs tophat and cufflinks.  Additional software programs, such as EdgeR, will be used to evaluate gene expression changes over time and/or between subjects.

Bioinformatics data will be organized using TACC’s iRODS metadata management layer, enabling embedded metadata with each data file to track relevant experimental and computational metadata.

#### QC Pipeline

_MRI_: Notes regarding scan quality will be recorded at scan time using Redcap.  MRI data will be assessed for quality by Dr. Mumford.  The automated quality control (fmriqa.py) tool will be run on each dataset.  Any problems with quality control will be reported to Dr. Luci who will investigate them and take whatever steps necessary to ameliorate them.

_Behavioral/lifestyle data_:  Dr. Simpson will do regular checks to ensure that the data are being properly recorded and logged to the database.

_RNA processing_: QC data for weekly RNA extraction will be entered into Redcap.

Sequencing:  TBD

**Analysis plan:**

_MRI_: Resting-state data will be modeled using the group ICA model (via concatenation).  Other measures will be compared to the ICA decomposition via dual regression.

Task based fMRI data will be analyzed on a weekly basis.  Each first level analysis will be run using Feat with our typical preprocessing steps: highpass filter at 100s, FILM prewhitening, 5mm smoothing and the model will use FILM prewhitening.

_Analysis over time._  The contrast estimates from the first level fMRI analyses will be correlated over time and we must adjust for this in our second level analyses.  Currently the plan is to test how well the FILM prewhitening can be used on cope based time series, in other words the first level contrast estimates over time.  Jeanette will try concatenating these and running a second level analysis using FILM to see if the prewhitening model, which uses a smoothed 1/frequency type covariance model.  This will be assessed through a residual analysis.  Jeanette will run voxelwise tests of the normality and correlation structure of the residuals as well as looking at qq plots of the residuals for randomly chosen voxels and averaged residuals within the regions of the Harvard Oxford atlas.  If a correlation structure remains in the residuals a secondary data analysis plan will be developed.

_Behavioral/lifestyle data_:  Jeanette will also run quality control on the behavioral data.

_Metabolic data_: TBD

_Transcriptome data_: TBD

#### Data sharing plan

All data except for genome sequence data will be shared publicly at the end of the data collection period, subject to an embargo period for data analysis prior to publication.  The data will be shared under a data use agreement that will restrict the disclosure of any health-related findings.
